{"name": "test_Login_002", "status": "broken", "statusDetails": {"message": "AttributeError: 'NoneType' object has no attribute 'get'", "trace": "self = <testCases.test_login.Test_CredKart_Login object at 0x00000254FCD524E0>\nsetup = None\n\n    def test_Login_002(self, setup):\n        self.driver = setup\n>       self.driver.get(\"https://automation.credence.in/login\")\nE       AttributeError: 'NoneType' object has no attribute 'get'\n\ntestCases\\test_login.py:23: AttributeError"}, "start": 1712483718095, "stop": 1712483718095, "uuid": "c7830465-2984-4cd7-b567-35bbde634f26", "historyId": "3860f7035c6b1319035d5e9677ae909b", "testCaseId": "3860f7035c6b1319035d5e9677ae909b", "fullName": "testCases.test_login.Test_CredKart_Login#test_Login_002", "labels": [{"name": "parentSuite", "value": "testCases"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "Test_CredKart_Login"}, {"name": "host", "value": "Nikhil"}, {"name": "thread", "value": "20216-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testCases.test_login"}]}